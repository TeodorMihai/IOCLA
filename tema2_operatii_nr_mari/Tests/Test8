CAFEBABE123434567344556722FABDEFFF1353 >> 1F
