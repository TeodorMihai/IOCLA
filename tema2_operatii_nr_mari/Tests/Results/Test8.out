5313FFEFBDFA226755447356343412BEBAFECA
1F
7BF545CEAA88E6AC6868247C75FD9501
195FD757C246868ACE688AACE45F57B
